compiler/optimizer/LocalAnalysis.cpp

```diff
@@ -548,6 +548,7 @@ int TR_LocalAnalysisInfo::hasOldExpressionOnRhs(TR::Node *node, bool recalcConta
       int32_t hashValue = _hashTable->hash(node);
       HashTable::Cursor cursor(_hashTable, hashValue);
       TR::Node *other;
+      bool tmpIsStoreIndirect = false;
```

Comment:
I think we should rename this to something more meaningful, such as `isOtherStoreIndirect` and put a comment to reference the comment which you will make for `isStoreIndirect` as this follows the same pattern.